src/string_object.cpp

```diff
@@ -1893,7 +1893,7 @@ Value StringObject::ljust(Env *env, Value length_obj, Value pad_obj) {
     }
 
     if (padstr->string().is_empty())
-        env->raise("ArgumentError", "can't pad with an empty string");
+        env->raise("ArgumentError", "zero width padding");
```

Comment:
Not relevant change, just fixed an error message (this message isn't checked in RubySpec, heh):

```ruby
# ruby 3.1.2p20

"a".ljust(10, "")
(irb):184:in `ljust': zero width padding (ArgumentError)
```